src/PostsIterator.php

```diff
@@ -35,7 +35,6 @@ public function current()
          */
         $wp_post = parent::current();
         // Lazily instantiate a Timber\Post instance exactly once.
-        // @todo maybe improve performance by caching the instantiated post.
```

Comment:
Performance and caching is an area where we could improve in a lot of places. In case we want to improve post performance, we’ll find the right place to do it.